lib/mongoose_push_behaviour.ex

```diff
@@ -0,0 +1,5 @@
+defmodule MongoosePushBehaviour do
+  @moduledoc false
+  @callback push(String.t(), MongoosePush.request()) ::
```

Comment:
This spec is incorrect, the input is different from what controller actually passes. This module should define the input type according to want controller will / is passing.

Reply:
done